pdf/en.norm.tex

```diff
@@ -123,7 +121,7 @@ \chapter{The Norm}
 
                 \item A line can never end with spaces or tabulations.
 
-                \item You can never have two consecutive spaces.
+                \item You can never have two consecutive spaces outside of tabulations.
```

Comment:
This wording isn't clear: tabulation is 4-space wide, but it isn't 4 spaces. Can lead to misunderstanding.

Reply:
You're right, I'm not sure why that slipped my mind at the time. I've reverted the wording back to its original form.